faros_event.sh

```diff
@@ -356,6 +459,13 @@ function processEventTypes() {
         addArtifactToData
         addCommitToData
         addRunToData
+    elif ((test_execution_event)); then
+        event_type="TestExecution"
```

Comment:
I think in the API you are expecting `testexecution`

Reply:
We lowercase the event in the API so I thought `TestExecution` was the most readable variant for the CLI to output.

Reply:
sgtm